chat-microservice/app/api/v1/messaging.py

```diff
@@ -44,21 +45,13 @@ async def create_message(
         )
     if not Chat.users_id_belongs_to_chat(message.chat_id, message.from_user, message.to_user):
         raise HTTPException(
-            status_code=401, detail=settings.CASSANDRA_MESSAGE_CREATION_UNAUTHORIZED
+            status_code=401, detail='users dont belong to any chat'
```

Comment:
Please keep it as main branch `settings.CASSANDRA_MESSAGE_CREATION_UNAUTHORIZED`